### ISSUE TITLE
LogEvent should implement toString()

### ISSUE THREAD

**Issue by mjsztainbok** [Hakky54/log-captor#33]
**Is your feature request related to a problem? Please describe.**
Currently `LogEvent` does not have a `toString()` method implementation. As such, when they are report in assertion failures they should the class name and reference only which isn't useful e.g.
```
Collection should have size 1 but has size 2. Values: [nl.altindag.log.model.LogEvent@56b58bd6, nl.altindag.log.model.LogEvent@8301464]
```

**Describe the solution you'd like**
For a `toString()` method to be implemented that outputs the contents of the `LogLevel` object

**Describe alternatives you've considered**
None as there are no sutiable alternatives

**Additional context**

**Comment by Hakky54** [comment]
Hi Mark,

This sounds as a nice idea. Just created a PR here https://github.com/Hakky54/log-captor/pull/34, please let me know if this would work out for you.

**Comment by Hakky54** [comment]
Hi @mjsztainbok what do you think of the Pull Request?

**Comment by mjsztainbok** [comment]
Looks good. The throwable will only show the type and message, right?

**Comment by Hakky54** [comment]
> The throwable will only show the type and message, right?

Yes, indeed. Just made a new release, good luck with it!. It will be available at version 2.10
